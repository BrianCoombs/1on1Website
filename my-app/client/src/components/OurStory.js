import React, { Component } from 'react'

class OurStory extends Component {


  render() {

    return (
      <div class="ourStory" id="ourstory">
        <br/><br/><br/>
        <h1 >Our Story</h1>

        <section class="timeline">
          <ul>

            <li>
              <div>
                <time>Jan 2011</time> At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium
              </div>
            </li>

            <li>
              <div>
                <time>October 2012</time> Proin iaculis, nibh eget efficitur varius, libero tellus porta dolor, at pulvinar tortor ex eget ligula. Integer eu dapibus arcu, sit amet sollicitudin eros.
              </div>
            </li>

            <li>
              <div>
                <time>May 2014</time> In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
              </div>
            </li>

            <li>
              <div>
                <time>November 2014</time> Aenean condimentum odio a bibendum rhoncus. Ut mauris felis, volutpat eget porta faucibus, euismod quis ante.
              </div>
            </li>

            <li>
              <div>
                <time>October 2015</time> Vestibulum porttitor lorem sed pharetra dignissim. Nulla maximus, dui a tristique iaculis, quam dolor convallis enim, non dignissim ligula ipsum a turpis.
              </div>
            </li>



          </ul>
        </section>


        <br/>
      </div>
    );
  }
}

export default OurStory;
