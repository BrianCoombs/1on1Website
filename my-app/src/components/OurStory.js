import React, { Component } from 'react'

class OurStory extends Component {

  render() {
    return (
      <div class="ourStory" id="ourstory">
        <br/>
        <h1 >Our Story</h1>
        <section class="section intro">
          <div class="container">
          </div>
        </section>

        <section class="timeline">
          <ol>
            <li>
              <div>
                <time>01/01/2010</time> In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
              </div>
            </li>
            <li>
              <div>
                <time>02/03/2011</time> In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
              </div>
            </li>
            <li>
              <div>
                <time>03/03/2013</time> In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
              </div>
            </li>
            <li>
              <div>
                <time>03/03/2014</time> In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
              </div>
            </li>
            <li>
              <div>
                <time>03/03/2015</time> In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
              </div>
            </li>
            <li>
              <div>
                <time>01/03/2016</time> In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
              </div>
            </li>
            <li></li>
          </ol>
        </section>
      </div>
    );
  }
}

export default OurStory;
