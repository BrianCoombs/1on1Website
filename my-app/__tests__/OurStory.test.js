import React from 'react'
import { configure, shallow } from 'enzyme'
import OurStory from '../src/components/OurStory'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'

configure({adapter: new Adapter()});

it('renders correctly', () => {
  const component = shallow(
    <OurStory />
  );
  expect(toJson(component)).toMatchSnapshot();
});