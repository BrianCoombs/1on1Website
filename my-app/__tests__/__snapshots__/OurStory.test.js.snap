// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="ourStory"
  id="ourstory"
>
  <br />
  <h1>
    Our Story
  </h1>
  <section
    class="section intro"
  >
    <div
      class="container"
    />
  </section>
  <section
    class="timeline"
  >
    <ol>
      <li>
        <div>
          <time>
            01/01/2010
          </time>
           In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </div>
      </li>
      <li>
        <div>
          <time>
            02/03/2011
          </time>
           In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </div>
      </li>
      <li>
        <div>
          <time>
            03/03/2013
          </time>
           In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </div>
      </li>
      <li>
        <div>
          <time>
            03/03/2014
          </time>
           In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </div>
      </li>
      <li>
        <div>
          <time>
            03/03/2015
          </time>
           In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </div>
      </li>
      <li>
        <div>
          <time>
            01/03/2016
          </time>
           In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </div>
      </li>
      <li />
    </ol>
  </section>
</div>
`;
